src/docfx/lib/YamlUtility.cs

```diff
@@ -130,18 +133,18 @@ private static JToken ToJson(YamlNode node, JTokenSourceMap mappings = null)
                     }
                     if (long.TryParse(scalar.Value, out var n))
                     {
-                        return SetMappings(mappings, scalar, new JValue(n));
+                        return new JValue(n);
                     }
                     if (double.TryParse(scalar.Value, out var d))
                     {
-                        return SetMappings(mappings, scalar, new JValue(d));
+                        return new JValue(d);
                     }
                     if (bool.TryParse(scalar.Value, out var b))
                     {
-                        return SetMappings(mappings, scalar, new JValue(b));
+                        return new JValue(b);
```

Comment:
lineinfo should be set to every token including JValue

Reply:
JValue is taken care of, when it returns to JObject call

---
In reply to: [203930758](https://github.com/dotnet/docfx/pull/3100#discussion_r203930758) [](ancestors = 203930758)